packaging/rpm/ansible-pylibssh.spec

```diff
@@ -62,6 +62,8 @@ Source24: %{pypi_source tomli 1.2.3}
 BuildRequires: openssh
 # sshd?
 BuildRequires: openssh-server
+# ssh?
+BuildRequires: openssh-clients
```

Comment:
Could you remind me why this is needed?

Reply:
Hi, because https://github.com/ansible/pylibssh/commit/376fc5215cfd04ebbd2ecf53128778783fe7ed8e

Reply:
https://github.com/ansible/pylibssh/commit/376fc5215cfd04ebbd2ecf53128778783fe7ed8e#diff-9417337bb13869644ad69cf16fc956100d4729cb2e25809e4a48cc86462760dbR48